python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/metadata.schema.yml

```
@@ -355,3 +355,21 @@ keys:
                           type: str
                         category:
                           type: str
+      digital_twin:
+        type: dict
+        description: Metadata used to generate Digital Twin topology.
+        keys:
+          environment:
+            type: str
+            description: Targeted Digital Twin environment (ACT, etc.).
```

Let's do valid_values here. Also consider adding "required" to all fields that are actually required. It will help us later, when we refactor fabric documentation to use the classes.

```suggestion
            description: Targeted Digital Twin environment.
            valid_values:
              - act
```

---

Fixed